proposals/extensions.md

```diff
@@ -178,19 +178,17 @@ public static class NullableExtensions
 }
 ```
 
-### Compatible extension methods
+### Compatibility with classic extension methods
 
-By default, extension members are lowered in such a way that the generated artifacts are not visible at the language level.
-However, if the receiver specification is in the form of a parameter and specifies the `this` modifier, 
-then any extension instance methods in that extension declaration will generate visible classic extension methods.
+Instance extension methods are lowered in such a way that the generated artifacts match those produced by classic extension methods.
```

Comment:
I think the "lowered" term is somewhat vague/undefined, unless it is explained in details later.